src/libraries/Microsoft.Extensions.Configuration.Xml/Microsoft.Extensions.Configuration.Xml.sln

```diff
@@ -57,10 +39,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.Security.Cryptograph
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{AC596411-85F4-49FD-8A23-BE2983825CF1}"
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ref", "ref", "{AB0BAC3A-1FE6-4649-83DF-DC165669C74F}"
-EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{373E0EA7-D9A1-4D7A-A300-9B2583AA09FD}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ref", "ref", "{AB0BAC3A-1FE6-4649-83DF-DC165669C74F}"
```

Comment:
Isn't the `ref` project being deleted?

Reply:
This is just the ref folder inside VS.

Reply:
Why would we need the ref folder in VS if the ref project is being deleted?

Reply:
Other reference project are still existent in the dependency graph of Microsoft.Extensions.Configuration.Xml.

Reply:
Ah, got it.